src/storage/mutate/UpdateEdgeProcessor.h

```diff
@@ -67,7 +73,8 @@ class UpdateEdgeProcessor
     std::string                                                     val_;
     std::unique_ptr<RowUpdater>                                     updater_;
     meta::IndexManager*                                             indexMan_{nullptr};
-    std::vector<std::shared_ptr<nebula::cpp2::IndexItem>>                            indexes_;
+    std::vector<std::shared_ptr<nebula::cpp2::IndexItem>>           indexes_;
+    FilterResult                                          filterResult_{FilterResult::FAILED_ERROR};
```

Comment:
Use atomic here,  also there is no problem because you only read the filterResult_ when ERR_ATOMIC_OP_FAILED. 
But it is not a good example for others.